run_test.py

```diff
@@ -62,7 +62,7 @@
         conf = {
             'base': 'ubuntu14_py2',
             'cuda': 'cuda65',
-            'cudnn': 'cudnn2',
+            'cudnn': 'cudnn5',
```

Comment:
I agree with you. For now, it is OK to change the combination to be tested as you do.